src/components/rpc_base/test/CMakeLists.txt

```diff
@@ -31,9 +31,8 @@
 if (BUILD_TESTS)
```

Comment:
@Kozoriz Whether it still should be there?
Please, check it out in every test `CMakeList.txt` file you have changed.